thirdparty/patches/librdkafka.patch

```diff
@@ -1,6 +1,6 @@
 --- src/rdkafka_broker.c
 +++ src/rdkafka_broker.c
-@@ -5388,7 +5388,7 @@ static int rd_kafka_broker_thread_main(void *arg) {
+@@ -4616,7 +4616,7 @@ static int rd_kafka_broker_thread_main(void *arg) {
```

Comment:
![image](https://user-images.githubusercontent.com/16649269/219571874-4a1cb4e2-cb20-48d0-93ee-d1d8610e2e2b.png)

Is the line number right?

Reply:
![image](https://user-images.githubusercontent.com/11428742/219572149-f88ea843-57c1-40ca-9daa-8df40b4b6606.png)

Reply:
Fixed